readme.md

```diff
@@ -60,6 +60,35 @@ Type: `RegExp`
 
 Regex to match against the module IDs.
 
+### clearModule.single(moduleId)
+
+Clear one module from cache non-recursively
+
+#### moduleId
+
+Type: `string`
+
+What you would use with `require()`.
+
+<details>
+<summary>Example use</summary>
+"clear" function will clear only the business logic of the module, any child dependencies will remain in memory.
+
+
+
+In the following example, `stats` module will **not** be cleared from memory
```

Comment:
This takes care of the **what**, but not the **why** or **when** you would want to use this method.

Reply:
How's this?
> This use case is relevant for systems sustaining a state that's dependant on the Singelton nature of modules. When user would want to clear a specific module from memory without it having any "side effects" such as clearing a child module from memory as well.

(This is also what I understood from the issue)

Reply:
Almost perfect.

> This use case is relevant for systems sustaining a state that's dependant on the Singelton nature of modules.

I think this could be written simpler. No need for "sustaining" and "nature of modules".

---

Also don't forget to add this to the TS file too. The readme and TS file should be in sync.

Reply:
:pencil: